src/polyfills.ts

```diff
@@ -8,64 +8,45 @@
  *      file.
  *
  * The current setup is for so-called "evergreen" browsers; the last versions of browsers that
- * automatically update themselves. This includes Safari >= 10, Chrome >= 55 (including Opera),
- * Edge >= 13 on the desktop, and iOS 10 and Chrome on mobile.
+ * automatically update themselves. This includes recent versions of Safari, Chrome (including
```

Comment:
It saddens me to say that Angular browser support may be ahead of gov.uk requirements. https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in-from-june-2021

Reply:
@premghinde - is this a concern for us as VTM won't be a GOV.UK externally facing service?

Reply:
Hurrah! I've found that all internal users have Edge as the default browser within DVSA, so it pleases me greatly to say you can indeed lose the legacy support.

Reply:
Thanks @premghinde, that's made my day!

Reply:
Believe me, the last thing I would have wanted to get you to do was reintroduce the legacy support.